source/_UIAHandler.py

```diff
@@ -61,6 +61,7 @@
 	# #7497: Windows 10 Fall Creators Update has an incomplete UIA implementation for console windows, therefore for now we should ignore it.
 	# It does not implement caret/selection, and probably has no new text events.
 	"ConsoleWindowClass",
+	"FoxitDocWnd",

```

Comment:
Could you put a comment above this alone? Something like:

`# #8944: The Foxit UIA implementation is incomplete and should not be used for now.`